Makefile

```diff
@@ -1,26 +1,13 @@
-# Copyright 2017 The Kubernetes Authors.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-PLATFORM       ?= linux/amd64
 REGISTRY_NAME  ?= index.docker.io/linode
 IMAGE_NAME     ?= linode-blockstorage-csi-driver
-REV            := $(shell git describe --long --tags --dirty)
+REV            := $(shell git describe --long --tags --dirty 2> /dev/null || echo "dev")
 IMAGE_VERSION  ?= $(REV)
 IMAGE_TAG      ?= $(REGISTRY_NAME)/$(IMAGE_NAME):$(IMAGE_VERSION)
 GOLANGCI_LINT_IMG := golangci/golangci-lint:v1.52-alpine
+RELEASE_DIR    ?= release
 
-export GO111MODULE=on
+.PHONY: ci
+ci: fmt vet lint test build
```

Comment:
I'm wondering if we should version golang ci lint settings to make the linting step more explicit. I mean... we pin the tool's version, so it's not like it's not repeatable, but it's largely unconfigured. (Probably out of scope for this PR since we already run this in CI in the current state.)

Reply:
i think that's a great idea... but agree it should be another PR. i vote we drop the `fmt` as discussed in the comment below and start a linter updates pr